.prettierignore

```diff
@@ -0,0 +1,11 @@
+bin
+docs
+lib
+node_modules
+reports
+tmp
+
+CHANGELOG.md
+CONTRIBUTING.md
+README.md
+UPGRADING.md
```

Comment:
Think we do probably want to extend to running prettier on these md files and most of docs. But thinking can enable that in a follow up

Reply:
Makes sense. We'd need to watch out for `CHANGELOG.md` as that gets written by a robot.